src/ui/input/text/text.module.scss

```diff
@@ -0,0 +1,37 @@
+.input {
+  outline: 0;
+  padding: 11px 16px;
```

Comment:
no sizes in default. use em, or extract to presets file

Reply:
please make a composition of 3 different sizes. we never have just one one size!

Reply:
done